sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage/CHANGELOG.md

```diff
@@ -2,6 +2,36 @@
 
 ## 5.0.0-beta.4 (Unreleased)
 
+### Major changes and features
+
+The `Microsoft.Azure.WebJobs.Extension.Storage` relies on storage providers in the WebJobs SDK to create BlobServiceClients and QueueServiceClients.
+- Support for running Functions with Managed Identity and secretless configurations
+- The new dependency adds a new configuration format to create Storage clients:
```

Comment:
I think we should move/copy this description to blobs and queues parts. This package is empty metapackage which might get dropped some day. I'd rather keep referencing blobs and queues changelogs here rather than being verbose here. (btw. he readme of this package encourages to use split extension packages).